webapp/init.sql

```diff
@@ -1,5 +1,5 @@
 
-INSERT INTO g_client (gc_name, gc_description, gc_client_id) VALUES ('admin app', 'Glewlwyd administration app', 'g_admin');
+INSERT INTO g_client (gc_name, gc_description, gc_client_id, gc_client_password) VALUES ('admin app', 'Glewlwyd administration app', 'g_admin', PASSWORD('password'));
```

Comment:
I'm not willing to make this change, because I see some problems with it:
- The client `admin app` is not supposed to be confidential, so there is no password required for it
- In the contrary, if you set `admin app` to confidential mode, its password will be required for every oauth2 token request, but the front-end application doesn't handles passwords for the client since it's a security leak.
- If you want to set another client to confidential mode, you also must set the column `gc_confidential` to `1`, otherwise the password will never be checked